llvm/include/llvm/IR/Intrinsics.td

```diff
@@ -1010,6 +1010,7 @@ let IntrProperties = [IntrNoMem, IntrSpeculatable, IntrWillReturn] in {
   def int_powi : DefaultAttrsIntrinsic<[llvm_anyfloat_ty], [LLVMMatchType<0>, llvm_anyint_ty]>;
   def int_sin  : DefaultAttrsIntrinsic<[llvm_anyfloat_ty], [LLVMMatchType<0>]>;
   def int_cos  : DefaultAttrsIntrinsic<[llvm_anyfloat_ty], [LLVMMatchType<0>]>;
+  def int_tan  : DefaultAttrsIntrinsic<[llvm_anyfloat_ty], [LLVMMatchType<0>]>;
```

Comment:
Adding `@llvm.tan` is a significant change; if you need this, please post a proposal on Discord explaining what you're adding an why.

Reply:
(Sorry, accidentally wrote Discord instead of Discourse initially.)

Reply:
Thanks. I will pull the pr for now as I investigate alternatives.